app/javascript/crayons/Buttons/Button.jsx

```diff
@@ -0,0 +1,62 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import classNames from 'classnames/bind';
```

Comment:
So this is to simplify setting classes. So far we've been using a [custom function](https://github.com/forem/forem/blob/a0744f3c99dbff8f85e6d1f2c4b84fd7c61cebab/app/javascript/crayons/Button/Button.jsx#L6-L46) but it felt a little dirty to me.

Reply:
All good. I've used this on other projects.